docs/SPEC.md

```diff
@@ -729,29 +729,45 @@ It is not possible to access an object's property using an arbitrary expression.
     DotExpression           = "." identifer
     MemberBracketExpression = "[" string_lit "]" .
 
+#### Conditional Expressions
```

Comment:
I think we should explicitly state that only one branch is evaluated. This only matters for side effects but we should be explicit about it.